ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activityQuestion.component.ts

```diff
@@ -18,9 +18,11 @@ import { delay, filter, map, shareReplay, startWith, takeUntil, tap } from 'rxjs
                   [validationRequested]="validationRequested$ | async"
                   (valueChanged)="enteredValue$.next($event)">
           </ddp-activity-answer>
-          <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
-              <ddp-validation-message [message]="errorMsg">
-              </ddp-validation-message>
+          <div *ngIf="block.shown">
+            <div class="ddp-activity-validation" *ngIf="errorMessage$ | async as errorMsg">
```

Comment:
@MocanaAtBroad I tried your suggestion of putting `block.shown` in here but I'm getting TypeScript compile errors. I lack the experience to debug this. So I think I'm going to proceed with my current approach as-is.

![Screen Shot 2021-01-11 at 12 27 24 PM](https://user-images.githubusercontent.com/5713833/104216799-88693580-5408-11eb-894c-6cf753026b51.png)